<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BFRuby</class>
 <widget class="QMainWindow" name="BFRuby">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bokra Fekra</string>
  </property>
  <property name="windowIcon">
   <iconset resource="BF_APP.qrc">
    <normaloff>:/icons/images/icon8.png</normaloff>:/icons/images/icon8.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="groupLabel">
      <property name="text">
       <string>&amp;Group:</string>
      </property>
      <property name="buddy">
       <cstring>groupComboBox</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="groupComboBox">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>784</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="membersLabel">
      <property name="text">
       <string>&amp;Members:</string>
      </property>
      <property name="buddy">
       <cstring>membersTable</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QTableView" name="membersTable"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>952</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menu_New_member">
     <property name="title">
      <string>&amp;New</string>
     </property>
     <property name="icon">
      <iconset resource="BF_APP.qrc">
       <normaloff>:/icons/images/icon4.png</normaloff>:/icons/images/icon4.png</iconset>
     </property>
     <addaction name="newMemberAction"/>
     <addaction name="newGroupAction"/>
    </widget>
    <addaction name="menu_New_member"/>
    <addaction name="manageMembersAction"/>
    <addaction name="separator"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="manageMembersAction">
   <property name="icon">
    <iconset resource="BF_APP.qrc">
     <normaloff>:/icons/images/icon1.png</normaloff>:/icons/images/icon1.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Manage members</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="icon">
    <iconset resource="BF_APP.qrc">
     <normaloff>:/icons/images/icon5.png</normaloff>:/icons/images/icon5.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="newMemberAction">
   <property name="icon">
    <iconset resource="BF_APP.qrc">
     <normaloff>:/icons/images/icon2.png</normaloff>:/icons/images/icon2.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Member</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="newGroupAction">
   <property name="icon">
    <iconset>
     <selectedon>:/icons/images/icon3.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Group</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="icon">
    <iconset>
     <selectedon>:/icons/images/icon7.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;About Bokra Fekra Course App</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="BF_APP.qrc"/>
 </resources>
 <connections/>
</ui>
